src/Agent/NewRelic/Agent/Core/Errors/ErrorService.cs

```diff
@@ -54,6 +54,50 @@ public bool ShouldIgnoreHttpStatusCode(int statusCode, int? subStatusCode)
             return ShouldIgnoreError(GetFormattedHttpStatusCode(statusCode, subStatusCode));
         }
 
+        private bool IsErrorExpected(Exception exception)
```

Comment:
nit: let's move private methods below the public contract.

Ideally we should have something like stylecop prevent this.